adserver/tests/test_tasks.py

```diff
@@ -70,6 +72,79 @@ def test_calculate_publisher_ctrs(self):
         self.publisher.refresh_from_db()
         self.assertEqual(self.publisher.sampled_ctr, 20)
 
+    def test_daily_update_keywords(self):
+        # Keyword Aggregation requires some targeting
+        self.flight.targeting_parameters = {
+            "include_keywords": ["django"],
```

Comment:
Given the way we sum, would the result be changed if the targeting was `"python"`?

Reply:
We generate a different record for each one. Since all the ads were for both, the data will be duplicated.